prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -6,9 +6,21 @@ postsubmits:
     spec:
       containers:
       - image: gcr.io/gcp-guest/osconfig-agent-build:latest
-        imagePullPolicy: Always
         command:
-        - "/main.sh"
+        - "/daisy_build_upload.sh"
+        env:
+        - name: PROJECT
+          value: "gcp-guest"
+        - name: ZONE
+          value: "us-central1-c"
+        - name: BASE_REPO
+          value: "GoogleCloudPlatform"
+        - name: REPOSITORY
+          value: "osconfig"
+        - name: POST_SUBMIT_JOB
```

Comment:
i think this may also be a prow env variable

Reply:
(probably not for this one, but why you need such an env? Also I would suggest pass it via flags instead of global env vars as a general practice)

Reply:
removed unnecessary env variables that i introduced. a lot of them are already being provided by prow. using them as arguments